tests/framework-tests/core/js/FluidJSTests.js

```diff
@@ -230,6 +230,29 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         });
     });
 
+    fluid.tests.roundToDecimalTests = [
+        {num: 1.555, scale: 5, expected: 1.555},
```

Comment:
Please add negative versions of these values (e.g. -1.555) as well as testing a variety of other values as seen in the stackoverflow thread, e.g. 1.005, 1.5, 55.549, 54.9, -55.55, -55.551, -55.59 etc.